internal/storage/debug_bucket.go

```diff
@@ -229,3 +229,11 @@ func (b *debugBucket) DeleteObject(
 	err = b.wrapped.DeleteObject(ctx, req)
 	return
 }
+
+func (b *debugBucket) DeleteFolder(ctx context.Context, folderName string) (err error) {
```

Comment:
In Debug logs we can see only DeleteFolder call.

Reply:
deleteObject needs to be part of debug logs, otherwise it will be harder to debug any delete related issues.

Reply:
However, if we encounter an error in DeleteObject, we are already logging it with the prefix "DeleteObject: ". Additionally, it won't be possible to log the error if we call deleteObject directly from bucket_handle, as previously it was called from dir.go where the instance was of type bucket. This ensured that all wrappers were called, resulting in the error appearing in the debug_gcs logs. Now that the instance is of type bucketHandle, it will not invoke all the wrappers, and the error logging mechanism will not be triggered.

Reply:
However, if we encounter an error in DeleteObject, we are already logging it with the prefix "DeleteObject: ". Additionally, it won't be possible to log the error if we call deleteObject directly from bucket_handle, as previously it was called from dir.go where the instance was of type bucket. This ensured that all wrappers were called, resulting in the error appearing in the debug_gcs logs. Now that the instance is of type bucketHandle, it will not invoke all the wrappers, and the error logging mechanism will not be triggered.

Reply:
However, if we encounter an error in DeleteObject, we are already logging it with the prefix "DeleteObject: ". 

Additionally, it won't be possible to log the it in debug_gcs logs if we call deleteObject directly from bucket_handle, as previously it was called from dir.go where the instance was of type bucket. This ensured that all wrappers were called, resulting in the logs appearing in the debug_gcs logs. Now that the instance is of type bucketHandle, it will not invoke all the wrappers, and the error logging mechanism will not be triggered.

Reply:
As discussed offline added calls from dir.go itself